tests/discrete.test.ts

```diff
@@ -1,15 +0,0 @@
-import { expect, test } from "vitest";
-
-import { discreteSIR } from "./examples/discreteSIR";
-
-test("can do anything with our interface", () => {
```

Comment:
you can restore this test if we support deterministic RNG, but it looks like that never got added to the random-js package.  The idea is that we have a random object that always returns expectations, which makes these tests much easier to write